### PR TITLE
Fix tableID encoding

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/pingcap/blog-cn/pull/175
@kennytm PTAL

### CODE REVIEW

**Comment by CaitinChen** [comment]
@shenli PTAL

File: 2017-07-11-tidbinternal2.md [hunk 1]
```diff
@@ -81,31 +81,33 @@ TiDB allocates a `TableID` to each table, an `IndexID` to each index, and a `Row
 Each row of data is encoded into a Key-Value pair according to the following rule:
 
 ```
-Key: tablePrefix_rowPrefix_tableID_rowID
+Key: tablePrefix_tableID_rowSeperator_rowID
```
Comment (kennytm) [inline 2017-07-11-tidbinternal2.md]: Perhaps use `recordPrefixSep` to match line 106.

File: 2017-07-11-tidbinternal2.md [hunk 2]
```diff
@@ -127,17 +129,19 @@ Up to now, we’ve already covered how to map Table onto Key-Value. Here is one
...
+t10_r3 --> ["PD", "Manager", 30]
 ```
 
 In addition to Primary Key, this table also has an Index. Assume that the ID of Index is 1, its data is:
+
 ```
-t_i_10_1_10_1 —> null
-t_i_10_1_20_2 --> null
-t_i_10_1_30_3 --> null
+t10_1_10_1 --> null
```
Comment (kennytm) [inline 2017-07-11-tidbinternal2.md]: t10_i1_10_1

File: 2017-07-11-tidbinternal2.md (hunk 1 above)
Comment (CaitinChen) [reply 2017-07-11-tidbinternal2.md]: @kennytm ？

**Comment by CaitinChen** [comment]
@kennytm PTAL again

**Comment by CaitinChen** [comment]
@kennytm PTAL again. Thanks.

File: 2017-07-11-tidbinternal2.md [hunk 3]
```diff
@@ -81,31 +81,33 @@ TiDB allocates a `TableID` to each table, an `IndexID` to each index, and a `Row
...
 Key: tablePrefix_idxPrefix_tableID_indexID_indexColumnsValue
 Value: rowID
 
-The above encoding rule applies to Unique Index while it cannot create a unique Key for Non-unique Index. The reason is that the `tablePrefix_idxPrefix_tableID_indexID_` of an Index is the same. It’s possible that the `ColumnsValue of` multiple rows is also the same. Therefore, we’ve made some changes to encode the Non-unique Index:
+The above encoding rule applies to Unique Index while it cannot create a unique Key for Non-unique Index. The reason is that the `tablePrefix_tableID_indexPrefixSep_` of an Index is the same. It’s possible that the `ColumnsValue of` multiple rows is also the same. Therefore, we’ve made some changes to encode the Non-unique Index:
 
 ```
-Key: tablePrefix_idxPrefix_tableID_indexID_ColumnsValue_rowID
-Value：null
+Key: tablePrefix_tableID_indexPrefixSep_indexedColumnsValue
```
Comment (kennytm) [inline 2017-07-11-tidbinternal2.md]: Also missing indexID

**Comment by CaitinChen** [comment]
@kennytm Please approve it~

**Comment by CaitinChen** [comment]
@kennytm Thanks. PTAL again.

**Pull request merged**